### PR TITLE
Pytest added for function project_id

### PR DESCRIPTION
**Pull request by egenc**
Purpose of this PR is to add tests for the functionCloudos.process_workflow_list in CloudOS.

Test data:
```
tests/test_data/process_job_list_initial_json.json
```

Tests are made for checking if the function project_id working as intended:
```
test_job_project_id
```
To run the test:
```
python3 -m pytest -s -v
```
Expected Output:
image

### CODE REVIEW

**Review by dapineyro — CHANGES_REQUESTED** [review]
@egenc The function tested here, was already added here: https://github.com/lifebit-ai/cloudos-cli/pull/116 Please, close this PR.

File: tests/test_clos/test_project_id.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+from tests.functions_for_pytest import load_json_file
+
+INPUT_PROJECT = "tests/test_data/projects.json"
+INPUT_WORKFLOW = "tests/test_data/workflows.json"
+APIKEY = 'vnoiweur89u2ongs'
+CLOUDOS_URL = 'http://cloudos.lifebit.ai'
+WORKSPACE_ID = 'lv89ufc838sdig'
+PROJECT_NAME = "lifebit-testing"
+WORKFLOW_NAME = "nf-core-deepvariant"
+
```
Comment (dapineyro) [inline tests/test_clos/test_project_id.py:16]:
```suggestion


```

File: tests/test_clos/test_project_id.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    job = Job(apikey=APIKEY,
+        cloudos_url=CLOUDOS_URL,
+        workspace_id=WORKSPACE_ID,
+        cromwell_token=None,
+        project_name=PROJECT_NAME,
+        workflow_name=WORKFLOW_NAME)
+    # get mock response
+    project_id = job.project_id
+    # check the response
+    assert isinstance(project_id, str) and len(project_id) > 0
```
Comment (dapineyro) [inline tests/test_clos/test_project_id.py:57]:
```suggestion
    assert project_id == '1234bc123125'
```
I think it's better to check for the known value of the project_id. Also, I'd suggest to change it in `projects.json` to something more informative than 'XXX' (e.g.: '1234bc123125')

File: tests/test_data/projects.json [hunk 3]
```diff
@@ -0,0 +1,507 @@
+[
+    {
+        "_id": "XXX",
```
Comment (dapineyro) [inline tests/test_data/projects.json:3]:
```suggestion
        "_id": "1234bc123125",
```
Change it for something unique in the JSON to be more informative

**Review by dapineyro — CHANGES_REQUESTED** [review]
Thanks @egenc . Please, take a look at my suggestions when you have time.

File: tests/test_clos/test_project_id.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
+"""Pytest added for checking project_id"""
```
Comment (dapineyro) [inline tests/test_clos/test_project_id.py:1]:
```suggestion
"""Pytest added for checking Job.project_id"""
```
Not really required as the filename is quite self-explanatory.

**Review by dapineyro — CHANGES_REQUESTED** [review]
Thanks @egenc ! Only check that minor suggestion about the `docstring`

**Review by danielboloc — APPROVED** [review]
Looks good @egenc!

**Review by dapineyro — APPROVED** [review]
LGTM! :+1:

**Pull request merged**